src/PgNamed.hs

```diff
@@ -28,6 +29,7 @@ module PgNamed
 
          -- * Errors
        , PgNamedError (..)
+       , WithError
```

Comment:
Yes, I was thinking about it as well. Good point.
The reason I do export it is because the documentation for query functions won't have mystical `WithError`. Will do the renaming!